src/utils.c

```diff
@@ -384,3 +390,62 @@ date_bucket(PG_FUNCTION_ARGS)
 	bucketed = DirectFunctionCall2(timestamp_bucket, PG_GETARG_DATUM(0), converted_ts);
 	return DirectFunctionCall1(timestamp_date, bucketed);
 }
+
+/* Returns approximate period in microseconds */
+int64
+get_interval_period_approx(Interval *interval)
+{
+	return interval->time + (((interval->month * 30) + interval->day) * USECS_PER_DAY);
```

Comment:
Create a `#define` for `30` that explains what it is or use an existing define like `USECS_PER_DAY`

Reply:
Fixed